tests/test_save_load.py

```diff
@@ -70,21 +71,29 @@ def test_save_load(tmp_path, model_class):
     # Check
     model.save(tmp_path / "test_save.zip")
     del model
-    model = model_class.load(str(tmp_path / "test_save.zip"), env=env)
 
-    # check if params are still the same after load
-    new_params = model.policy.state_dict()
+    # Check if the model loads as expected for every possible choice of device:
+    for device in ["auto", "cpu", "cuda", th.device("cpu"), th.device("cuda")]:
```

Comment:
```suggestion
    for device in ["auto", "cpu", "cuda"]:
```
Although I agree the test would be more complete when using `th.device` variant,
the conversion from str to `th.device` is handled by `get_device()` and should not be tested here.
The test also run almost 2x faster that way.